### PR TITLE
Fixed #31577 -- Clarified docs about bounds of RangeFields.

### PR DESCRIPTION
**Pull request by xncbf**
[Ticket 31577](https://code.djangoproject.com/ticket/31577)

### CODE REVIEW

File: docs/ref/contrib/postgres/fields.txt [hunk 1]
```diff
@@ -599,7 +599,7 @@ excluded; that is, ``[)``.
 
     Regardless of the bounds specified when saving the data, PostgreSQL always
     returns a range in a canonical form that includes the lower bound and
-    excludes the upper bound; that is ``[)``.
+    excludes the upper bound; According to `RangeBoundary() expressions`_, that is ``'[)'``.
```
Comment (felixxm) [inline docs/ref/contrib/postgres/fields.txt:602]: IMO, we shouldn't mix fields descriptions with `RangeBoundary()` expression. `[)` is set according to the `PostgreSQL` implementation, not according to our expression. Maybe:

```
... that is ``[)``. The PostgreSQL documentation has more details about `bounds
<https://www.postgresql.org/docs/current/rangetypes.html#RANGETYPES-INCLUSIVITY>`_.
```

Comment (xncbf) [reply docs/ref/contrib/postgres/fields.txt:602]: I agree with you. This is not related to Django's RangeBoundary expression, so i'll fix it as you suggested.

Comment (xncbf) [reply docs/ref/contrib/postgres/fields.txt:602]: Btw, Is it intended end of Semicolon like this?
`excludes the upper bound; `

Comment (felixxm) [reply docs/ref/contrib/postgres/fields.txt]: It depends, in this case we can use a comma.

**Comment by felixxm** [comment]
@xncbf Thanks :+1: I pushed small edits.

**Comment by felixxm** [comment]
IMO, https://www.postgresql.org/docs/current/rangetypes.html#RANGETYPES-IO fits better and we don't have to repeat this reference for all `RangeField`'s

**Comment by xncbf** [comment]
Great! This code is perfect.

**Pull request merged**